lib/benchee/suite.ex

```diff
@@ -47,3 +47,93 @@ defimpl DeepMerge.Resolver, for: Benchee.Suite do
     Map.merge(original, override, resolver)
   end
 end
+
+if Code.ensure_loaded?(Table.Reader) do
+  defimpl Table.Reader, for: Benchee.Suite do
+    alias Benchee.CollectionData
+    alias Benchee.Scenario
+
+    def init(suite_results) do
+      columns = get_columns_from_suite(suite_results)
+      {rows, count} = extract_rows_from_suite(suite_results)
+
+      {:rows, %{columns: columns, count: count}, rows}
+    end
+
+    defp get_columns_from_suite(suite_results) do
+      config_percentiles = suite_results.configuration.percentiles
+
+      percentile_labels =
+        Enum.map(config_percentiles, fn percentile ->
+          "p_#{percentile}"
+        end)
+
+      fields_per_type =
+        [
+          "samples",
+          "ips",
+          "average",
+          "maximum",
+          "median",
+          "minimum",
+          "mode",
+          "sample_size",
+          "std_dev"
+        ] ++ percentile_labels
+
+      memory_fields = Enum.map(fields_per_type, fn field -> "memory_#{field}" end)
+      reductions_fields = Enum.map(fields_per_type, fn field -> "reductions_#{field}" end)
+      run_time_fields = Enum.map(fields_per_type, fn field -> "run_time_#{field}" end)
```

Comment:
Theoretically we'd only want to collect these if theses measurements have been run, right? We could use `Scenario.data_processed?` to check although I see that is missing the reductions implementation.

Or are these auto hidden if empty? I really don't know.